recipes/huffman/all/conanfile.py

```diff
@@ -0,0 +1,56 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class HuffmanConan(ConanFile):
+    name = "huffman"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/drichardson/huffman"
+    description = "huffman encoder/decoder"
+    topics = ["huffman", "conan"]
+    license = "Unlicense"
+    settings = "os", "compiler", "arch", "build_type"
+    options = {"shared": [True, False],
+               "fPIC": [True, False]}
+    default_options = {'shared': False,
+                       'fPIC': True}
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def configure(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+        if self.settings.os == "Windows" or self.options.shared:
+            del self.options.fPIC
```

Comment:
We remove `fPIC` in `config_options` such that configuring a library with `-o fPIC=True` on Windows will throw an error.
```suggestion
        return "source_subfolder"

    def config_options(self):
        if self.settings.os == "Windows":
            del self.options.fPIC

    def configure(self):
        del self.settings.compiler.libcxx
        del self.settings.compiler.cppstd
        if self.options.shared:
            del self.options.fPIC
```